### DISCUSSION TITLE
What's the setup process please?

### DISCUSSION THREAD

**Discussion by Cuthbert-Dibble-Grubb** [BottlecapDave/HomeAssistant-OctopusEnergy#457] [Q&A]
I've installed Octopus Energy with HACS and now I need to know how to set it up. The link in "Please follow the [setup guide](http://homeassistant.local:8123/_docs/setup_account.md) to setup your initial account" just takes me to the HA overview screen (http://homeassistant.local:8123/lovelace/default_view). When I paste the target URL into the browser bar I just get "404: Not Found".

Where can I find setup instructions please? Thanks.

**Comment by BottlecapDave** [comment]
The readme, like other HACS projects, don't work very well with relative paths within the HACS screen. The url you're after is https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md if you access it from https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy
